src/World.cpp

```diff
@@ -1005,9 +1005,12 @@ void cWorld::Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_La
 		for (auto & Entity : m_EntitiesToAdd)
 		{
 			Entity->SetWorld(this);
-			m_ChunkMap->AddEntity(Entity);
-			ASSERT(!Entity->IsTicking());
-			Entity->SetIsTicking(true);
+
+			auto EntityPtr = Entity.get();
+			m_ChunkMap->AddEntity(std::move(Entity));
+
+			ASSERT(!EntityPtr->IsTicking());
+			EntityPtr->SetIsTicking(true);
```

Comment:
Not really. Empty lines are good for separating logical pieces of code, but if every other line is empty, it's like they aren't there anymore.

This whole thing has one purpose - add the entity into the chunkmap, with all bells and whistles relevant to it. So it makes sense for it to be one logical block of code.